tmt/base.py

```diff
@@ -2406,7 +2401,7 @@ def __init__(self,
         # have not been processed yet and we do not want commands such as
         # tmt run discover --how fmf --help to create a new workdir.
         super().__init__(context=context, logger=logger)
-        self._workdir_path: WorkdirArgumentType = id_ or True
+        self._workdir_path: WorkdirArgumentType = True
```

Comment:
This is suspicious. Any idea why `id_` would no longer be an interesting value? It might become `Path` thanks to `id_ = self.config.last_run`.

Reply:
Nice catch! I've opened https://github.com/charliermarsh/ruff/issues/4058 and disabling SIM222 for now.